SimG4Core/Application/src/SteppingAction.cc

```diff
@@ -13,7 +13,9 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/isFinite.h"
 
-//#define DebugLog
+#include <DD4hep/Filter.h>
```

Comment:
The same here, better do not depend on external package - std should have methods to resolve such problems.